pkg/resource/pod-resource-manger.go

```diff
@@ -0,0 +1,204 @@
+package resource
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	info "github.com/google/cadvisor/info/v1"
+	"google.golang.org/grpc"
+	v1 "k8s.io/api/core/v1"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	coreinformers "k8s.io/client-go/informers/core/v1"
+	clientset "k8s.io/client-go/kubernetes"
+	corelisters "k8s.io/client-go/listers/core/v1"
+	"k8s.io/client-go/tools/cache"
+	pb "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+	"k8s.io/klog/v2"
+
+	"github.com/gocrane/crane/pkg/common"
+	"github.com/gocrane/crane/pkg/ensurance/collector/cadvisor"
+	"github.com/gocrane/crane/pkg/ensurance/collector/types"
+	stypes "github.com/gocrane/crane/pkg/ensurance/collector/types"
+	"github.com/gocrane/crane/pkg/ensurance/executor"
+	cgrpc "github.com/gocrane/crane/pkg/ensurance/grpc"
+	cruntime "github.com/gocrane/crane/pkg/ensurance/runtime"
+	"github.com/gocrane/crane/pkg/utils"
+)
+
+type PodResourceManager struct {
```

Comment:
pod-resource-manger.go => pod_resource_manger.go , the golang file name standard ？

Reply:
Ok